### PR TITLE
Added missing terminating newlines

### PR DESCRIPTION
**Pull request by oliverwillder**

### CODE REVIEW

**Comment by eregon** [comment]
Could you do it for all files which do not have a terminating new line?
You can easily get a list of those with:
```ruby
ruby -rpathname -e 'Pathname.glob("**/*.rb") { |f| puts f if f.read.chars.last != "\n" }'
```

**Comment by oliverwillder** [comment]
Yeah totally! :)

**Comment by oliverwillder** [comment]
@eregon should be fine now! :)

File: core/module/fixtures/path2/load_path.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (eregon) [inline core/module/fixtures/path2/load_path.rb]: This file should stay empty though, that's a bug of my script :wink:

**Comment by eregon** [comment]
I squashed your commits and removed the change for `core/module/fixtures/path2/load_path.rb`,
see c35351504e0504ef1c81ecb768d97af2c6b81543.
Thank you for your contribution!

**Pull request closed without merging**